src/vars/foreman.yml

```
@@ -2,3 +2,7 @@
 foreman_admin_passwd_file: "{{ obsah_state_path }}/foreman-admin-init-passwd"
 foreman_initial_admin_username: admin
 foreman_initial_admin_password: "{{ lookup('ansible.builtin.password', foreman_admin_passwd_file, chars=['ascii_letters']) }}"
+# Puma workers calculation:
+# CPU based calculation is based on https://github.com/puma/puma/blob/master/docs/deployment.md#mri
+# Memory based calculation is based on https://docs.gitlab.com/ee/install/requirements.html#puma-settings
+foreman_service_puma_workers: "{{ [100, (ansible_processor_vcpus * 1.5) | int, ((ansible_memtotal_mb / 1024) - 1.5) | int] | min }}"
```

In https://github.com/theforeman/foremanctl/pull/218/files#diff-5fb78252acba9a0f6406f470d587d4d8c84771561e8101b67e23ea2463f9ca38 we ended up using `ansible_facts.processor_nproc`, not `_vcpus`

Probably should use the same fact in both places.

Also, looking at the Puppet code and here, would this be more readable:
```suggestion
foreman_service_puma_workers: "{{ [100, ansible_processor_vcpus * 1.5, (ansible_memtotal_mb / 1024) - 1.5] | min | int }}"
```
(only calls `int` once on the resulting smallest value)